packages/renderer/src/App.svelte

```diff
@@ -85,59 +85,56 @@ window.events?.receive('display-troubleshooting', () => {
         <Route path="/" breadcrumb="Dashboard Page">
           <DashboardPage />
         </Route>
-        <Route path="/containers" breadcrumb="Containers">
+        <Route path="/containers" breadcrumb="Containers" navLevel="list">
```

Comment:
just wondering if we should not then have a `<RouteList>` ,`<RouteTab>`and `<RouteDetails>` elements ?

Reply:
or if it's possible to automatically detect that with the `meta` object because we should know what were the routings to go to that page ?